mlrose_hiive/opt_probs/max_k_color_opt.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, edges=None, length=None, fitness_fn=None, maximize=False,
         else:
             self.source_graph = source_graph
 
+        self.stop_fitness = self.source_graph.number_of_edges() if maximize else 0
```

Comment:
Add stopping criterion for max k-colour problem if the problem is a maximisation problem.